tests/pipeline/test_engine.py

```diff
@@ -8,13 +8,15 @@
 
 from nose_parameterized import parameterized
 from numpy import (
+    append,
```

Comment:
FWIW, the already-imported `np.concatenate` does the same thing that you're accomplishing with `append`, except it takes a list of arrays instead of two positional args.